digest.gemspec

```
@@ -26,6 +26,7 @@ Gem::Specification.new do |spec|
   if Gem::Platform === spec.platform and spec.platform =~ 'java' or RUBY_ENGINE == 'jruby'
     spec.platform = 'java'
 
+    spec.files += Dir["lib/*.jar"]
     spec.files += Dir["ext/java/**/*.{rb,java}"]
```

I think is  equivalent?

```suggestion
    spec.files += Dir["ext/java/**/*.{rb,java}", "lib/*.jar"]
```